cmd/keeper/cmd/keeper.go

```diff
@@ -811,18 +813,50 @@ func (p *PostgresKeeper) resync(db, followedDB *cluster.DB, tryPgrewind bool) er
 	// doesn't exists pgm.SyncFromFollowedPGRewind will return an error and
 	// fallback to pg_basebackup
 	if tryPgrewind && p.usePgrewind(db) {
-		connParams := p.getSUConnParams(db, followedDB)
-		log.Infow("syncing using pg_rewind", "followedDB", followedDB.UID, "keeper", followedDB.Spec.KeeperUID)
-		// TODO: Make the forceCheckpoint parameter use cluster specification
-		if err := pgm.SyncFromFollowedPGRewind(connParams, p.pgSUPassword, true); err != nil {
-			// log pg_rewind error and fallback to pg_basebackup
-			log.Errorw("error syncing with pg_rewind", zap.Error(err))
+		var err error
+		timeout := db.Spec.PgrewindTimeout.Duration
+		interval := db.Spec.PgrewindInterval.Duration
+		checkpoint := db.Spec.PgrewindCheckpoint
+
+		pgrewind := func() error {
```

Comment:
Not sure about the behaviour of this when the cluster changes. Does it update the